domain/src/main/java/org/oppia/android/domain/topic/StoryProgressController.kt

```diff
@@ -110,7 +112,10 @@ class StoryProgressController @Inject constructor(
         storyProgressBuilder.putChapterProgress(explorationId, chapterProgress)
         val storyProgress = storyProgressBuilder.build()
 
-        val topicProgressBuilder = TopicProgress.newBuilder().setTopicId(topicId)
+        val topicProgressBuilder =
+          TopicProgress.newBuilder()
+            .setTopicId(topicId)
+            .setLastPlayedTimestamp(oppiaClock.getCurrentCalendar().timeInMillis)
```

Comment:
I am not able to get this. By structure you mean from `TopicProgress` message I should remove the timeStamp field from there?
This timeStamp is set in recentlyPlayedChapter as well.

Reply:
@BenHenning this timestamp in topicProgress is used only in sorting the list. I couldn't come up with better solution to sort so I added timestamp in topicProgress. Otherwise, to sort the list I had to traverse to chapter progress to get timestamp.

Reply:
Fixed this by computing lastplayedTimestamp from chapterlist.